Taller152/src/paqutedos/Principal.java

```diff
@@ -5,11 +5,45 @@
  */
 package paqutedos;
 
+//import java.io.FileNotFoundException;
+import java.util.Locale;
+import java.util.Scanner;
+
 
 /**
  *
  * @author reroes
  */
 public class Principal {
-    
+     public static void main(String[] args) {
+        Scanner e = new Scanner(System.in);
+        e.useLocale(Locale.US);
+        boolean bandera = true;
+        String equipo;
+        String estadio;
+        String negacion;
+        String mensaje = "";
+        char x;
+        double presupuesto;
+        while (bandera) {
+
+            System.out.println("Ingrese el nombre del equipo: ");
+            equipo = e.nextLine();
+            System.out.println("Ingrese el presupuesto anual del equipo: ");
+            presupuesto = e.nextDouble();
+            e.nextLine();
+            System.out.println("Ingrese el estadio donde juega el equipo: ");
+            estadio = e.nextLine();
+            mensaje = String.format("%s%s, presupuesto %.2f, estadio %s\n", mensaje,
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.